portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+/** Class containing server statistics. */
+public final class Comment {
+  String name;
+  String message;
+  String timeCreated;
+  Long id;
+
+  public Comment(String name, String message,  String timestamp, Long id) {
+    this.name = name;
+    this.message = message;    
+    this.timeCreated = timestamp;
+    this.id = id;
+  }
+
+  public void setDate(String newDate) {
+    this.timeCreated = newDate;
+  }
```

Comment:
Why is this block necessary

Reply:
In an earlier implementation I was setting the date via this setter, but I found out I can use gson to directly convert response -> Comment object. Forgot to remove this after new implementation, thanks! Will resolve in next commit.